src/containers/Chat/ChatMessages/ChatMessages.tsx

```diff
@@ -328,10 +330,17 @@ export const ChatMessages: React.SFC<ChatMessagesProps> = ({ contactId }) => {
     <Container className={styles.ChatMessages} disableGutters>
       {dialogBox}
       {toastMessage}
-      <ContactBar contactName={conversations.contact.name} />
-      <Container className={styles.MessageList} data-testid="messageContainer">
-        {messageList}
-      </Container>
+      <ContactBar contactName={conversations ? conversations.contact.name : 'No contacts'} />
+      {/* To avoid warnings with empty children in Container */}
```

Comment:
I decided switching between element types would allow for greatest flexibility, as `Container` throws a warning when children are `undefined`.